detox/src/realms/DetoxPrimaryContext.test.js

```diff
@@ -1,62 +1,73 @@
-// @ts-nocheck
 const _ = require('lodash');
 
 const {
+  backupProcessEnv,
   latestInstanceOf,
   lastCallTo,
-  throwErrorImpl,
-  withSuspendingMock,
-  uuidRegexp,
-  tempFileRegexp,
-} = global;
-
-const workerId = 19;
+} = require('../../__tests__/helpers');
 
 describe('DetoxPrimaryContext', () => {
+  //#region Fixtures and constants
+  const UUID_REGEXP_STR = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}';
```

Comment:
I prefer to use UPPER_CASE for constant variables which are not being reset in `beforeEach()` and/or `afterEach()` hooks.

Reply:
Also, maybe it is a superstition, but I try not to declare variables out of the root describe.